src/Application.cpp

```diff
@@ -149,6 +150,11 @@ void Application::initialize(Settings &settings, Paths &paths)
         this->initNm(paths);
     }
     this->initPubSub();
+
+    if (this->twitch->seventvEventAPI)
+    {
+        this->initSeventvEventAPI();
+    }
```

Comment:
Don't check `this->twitch->seventvEventAPI` here, early out inside the function instead with a qcdebug and/or comment stating why it's happening (i.e. setting is disabled)